.github/workflows/build.yml

```diff
@@ -47,23 +48,23 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v1
+      - uses: actions/setup-python@v2
         with:
           python-version: 3.8
       - name: Store installed Python version
         run: |
           echo "::set-env name=PY_VERSION::"\
           "$(python -c "import platform;print(platform.python_version())")"
       - name: Cache pip test requirements
```

Comment:
I think this should be "Cache testing environments" to mirror the language changes made to the name of the `lint` cache step:
https://github.com/cisagov/guacamole-composition/blob/e7b557eb949d3a42cc194c2efdaa3f3cf06f9788/.github/workflows/build.yml#L26

Reply:
Changed in commit aa58a73.

Reply:
Thanks for taking the time!